client/blocks/reader-post-card/gallery.jsx

```diff
@@ -2,20 +2,26 @@
  * External Dependencies
  */
 import React from 'react';
-import { map, take } from 'lodash';
+import { map, take, filter } from 'lodash';
 
 /**
  * Internal Dependencies
  */
+import { imageIsBigEnoughForGallery } from 'state/reader/posts/normalization-rules';
 import resizeImageUrl from 'lib/resize-image-url';
 
-const GALLERY_ITEM_WIDTH = 206;
+const GALLERY_ITEM_THUMBNAIL_WIDTH = 420;
```

Comment:
this is the width of the thumb, not the width of the image in the gallery itself. Only really matter when there are fewer than 4 images in the gallery. Shouldn't happen though..